docs/UserGuide.md

```diff
@@ -330,7 +339,6 @@ NetConnect data are saved automatically as a JSON file `[JAR file location]/data
 If your changes to the data file makes its format invalid, NetConnect will discard all data and start with an empty data file at the next run.  Hence, it is recommended to take a backup of the file before editing it.<br>
 Furthermore, certain edits can cause the NetConnect to behave in unexpected ways (e.g., if a value entered is outside the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
 
-</section>
```

Comment:
is this belong to `line 333`?

Reply:
one of the issue i receive ask me to specify if I don't understand wrongly #174